Automattic/woocommerce-payments
# Update packages to fix high severity vulnerabilites

Fixes the 16418 high severity vulnerabilities detected by `npm audit`.

`npm` detected _many_ high severity vulnerabilites:

<img width="657" alt="image" src="https://user-images.githubusercontent.com/13835680/61548059-43935700-aa3c-11e9-8227-1ff53f32cca2.png">

After running `npm audit fix` there is only one vulnerability remaining, and that one needs manual review:

<img width="657" alt="image" src="https://user-images.githubusercontent.com/13835680/61548205-ad136580-aa3c-11e9-8ced-12d210edef11.png">

#### Changes proposed in this Pull Request

* Update vulnerable packages to patch security issues via `npm audit fix`.

#### Testing instructions



* `npm i` to update packages.
* `npm run watch` and make sure web app works as intended.
